.gitignore

```diff
@@ -3,3 +3,4 @@ coverage
 npm-debug.log
 lib
 .eslintcache
+.DS_Store
```

Comment:
I suggest to put this in a global gitignore on your machine. I'm not super keen on having platform-specific entries in the gitignore file.

Reply:
@gabro that makes sense. I removed `.DS_Store` from gitignore and updated the PR. Thanks!